### PR TITLE
Colaboratory를 통해 생성됨

### PR DESCRIPTION
**Pull request by kwonzero**
베이스라인 + cutmix + FocalLoss + image size 조정
Efficient-net B2, batch_size=32

result
train loss : 0.69266
validation loss : 0.66316
f1 score : 0.71821
epoch : 56

0.6760987442 -> Public 67위
0.6534061887 -> Private 44위

### CODE REVIEW

**Comment by kwonzero** [comment]
upload
